config/locales/client.en.yml

```diff
@@ -1464,6 +1464,11 @@ en:
       shift: "Shift"
       ctrl: "Ctrl"
       alt: "Alt"
+      enter: "Enter"
+      shift_lowercase: "shift"
```

Comment:
Do we really need the lowercase versions? AFAIK only Apple uses lowercase keys on their keyboards.

https://codekeyboards.com/galleria/8.jpg

Reply:
My keyboard (Razer) uses lowercase on the modifier keys 😛 

But I don't have any feelings on this; I simply added the lowercase versions because that's what the keyboard shortcuts modal uses at the moment. If it makes you uneasy I'm happy to remove it.

Reply:
Yeah, I strongly feel we shouldn't create duplicate translations for such a minor reason. I'd go with the existing translations and remove the lowercase versions.